src/MCPClient/osdeps/Ubuntu-14.json

```diff
@@ -0,0 +1,49 @@
+{
+  "_comments": [
+    "Add any comments here",
+    "Include here keys, repos, and packages required by this component not already in archivematicaCommon osdeps file"
+  ],
+  "osdeps_repokeys": [
+    { "id": "0x5236CA08",
+      "url": "https://packages.archivematica.org/GPG-KEY-archivematica",
+      "validate_certs": "no"
+    }
+  ],
+  "osdeps_repos": [
+    "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty multiverse",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty-security universe",
+    "deb http://archive.ubuntu.com/ubuntu/ trusty-updates multiverse"
+  ],
+  "osdeps_packages": [
+  { "name": "atool", "state": "latest"},
+  { "name": "bagit", "state": "latest"},
+  { "name": "bulk-extractor", "state": "latest"},
+  { "name": "ffmpeg", "state": "latest"},
```

Comment:
I also think that we should add a known working version as minimum, keeping things explicit helps in the long term.